egs/aishell/ASR/transducer_stateless/model.py

```diff
@@ -108,18 +108,18 @@ def forward(
         # Note: y does not start with SOS
         y_padded = y.pad(mode="constant", padding_value=0)
 
+        y_padded = y_padded.to(torch.int64)
+        boundary = torch.zeros(
+            (x.size(0), 4), dtype=torch.int64, device=x.device
+        )
+        boundary[:, 2] = y_lens
+        boundary[:, 3] = x_lens
+
         assert hasattr(torchaudio.functional, "rnnt_loss"), (
             f"Current torchaudio version: {torchaudio.__version__}\n"
             "Please install a version >= 0.10.0"
         )
 
-        loss = torchaudio.functional.rnnt_loss(
-            logits=logits,
-            targets=y_padded,
-            logit_lengths=x_lens,
-            target_lengths=y_lens,
-            blank=blank_id,
-            reduction="sum",
-        )
+        loss = k2.rnnt_loss(logits, y_padded, blank_id, boundary)
 
-        return loss
+        return torch.sum(loss)
```

Comment:
Just to make you know that, the default reduction for `k2.rnnt_loss` is "mean" (the same as pytorch rnnt_loss), so you don't need a `torch.sum` here.

Reply:
I thought we were summing the loss??
If not, surely the normalization by "frames" (which we do in the diagnostics) would be wrong.

Reply:
We used reduction=="sum" in our expts.

Reply:
The returned `loss` is a scalar if `reduction=sum` is used, so we should change `return torch.sum(loss)` to `return loss`, even though `return torch.sum(loss)` is equivalent to `return loss` when `loss` is a scalar.

Reply:
Yes,  torch.sum is unnecessary here, will fix it. thanks